src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -483,4 +483,16 @@
   <data name="Show_completion_list_after_a_character_is_deleted" xml:space="preserve">
     <value>Show completion list after a character is _deleted</value>
   </data>
+  <data name="brace_preferences_colon" xml:space="preserve">
+    <value>brace preferences:</value>
+  </data>
+  <data name="Everywhere" xml:space="preserve">
+    <value>Everywhere</value>
+  </data>
+  <data name="Prefer_braceless" xml:space="preserve">
+    <value>Prefer braceless</value>
```

Comment:
I'm not sure if this is an actual word in the english language :) eitherway, can someone confirm if this wording is okay?